Watchman.DomainModel/Users/Commands/Handlers/MarkMuteEventAsUnmutedCommandHandler.cs

```diff
@@ -17,7 +17,7 @@ public async Task HandleAsync(MarkMuteEventAsUnmutedCommand command)
         {
             using var session = this._sessionFactory.Create();
             var muteEvent = session.Get<MuteEvent>(command.MuteEventGuid);
-            muteEvent.Unmuted = true;
+            muteEvent.IsUnmuted = true;
```

Comment:
takich rzeczy nie możemy robić to to obiekt domenowy
aż dziwne że takie coś jest możliwe, musiałem czegoś nie zauważyć...

do tego powinny być odpowiednie metody
np 
```csharp
public void Unmute()
{
    if(this.IsUnmuted)
    {
        return;
    }
    this.IsUnmuted = true;
    this.Update()
}
```
kluczowe jest to `this.Update`

Reply:
i każdy obiekt domenowy powinien mieć takie metody 
nie mają bo wyszliśmy z założenia `doda sie jak będą potrzebne`

Reply:
to powinno być w samej klasie MuteEvent?